calibration/DistributionUtils.jl

```diff
@@ -0,0 +1,29 @@
+import EnsembleKalmanProcesses as EKP
+
+function initial_parameter_ensemble(priors, N_ensemble; rng_seed = 10)
+    Random.seed!(rng_seed)
+    initial_ensemble = EKP.construct_initial_ensemble(priors, N_ensemble)
+    return initial_ensemble
+end
+
+function construct_priors(
+    params::Dict{String, @NamedTuple{mean::FT, var::FT, lbound::FT, ubound::FT}},
+) where {FT <: Real}
+    u_names = collect(keys(params))
+    u_values = collect(values(params))
+    n_params = length(u_names)
+
+    # All vars are approximated as Gaussian in unconstrained space
+    marginal_priors = Vector{EKP.ParameterDistribution}(undef, n_params)
+    for i in 1:n_params
+        marginal_priors[i] =
```

Comment:
I see these lines are (slightly) different from how they looked like in the original file. This PR would be easier to review if it only moved files and did the minimal edits to make imports work correctly, then make a new PR where you do various bike shedding.

Reply:
Or alternatively, make changes to import statements in PR 1, then move files in PR 2.